File: app/src/main/java/com/saatco/murshadik/models/Plant.java
package com.saatco.murshadik.models;

public class Plant {
    private String name;
    private int id;

    public Plant(String name, int id) {
        this.name = name;
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    // Override toString() to display the name in the Spinner
    @Override
    public String toString() {
        return name;
    }
}

File: app/src/main/java/com/saatco/murshadik/models/Plant.kt
package com.saatco.murshadik.models

import com.google.gson.annotations.SerializedName

data class Plant(
    val id: Int,
    @SerializedName("english_name")
    val englishName: String,
    @SerializedName("arabic_name")
    val arabicName: String
)

data class PlantListResponse(
    val items: List<Plant>
)